dashboard-visualiser-kibana/package-metadata.json

```diff
@@ -6,8 +6,8 @@
   "version": "0.0.1",
   "dependencies": ["analytics-datastore-elastic-search"],
   "environmentVariables": {
-    "STATEFUL_NODES": "single",
     "KIBANA_INSTANCES": 1,
```

Comment:
Didn't you define CLUSTERED_MODE for every package metadata file ?

Reply:
actually, CLUSTERED_MODE is in reality not an env var, we discussed that before.
Brad mentioned that  in prev comment so that I removed it from all the package metadata files